tests/system/robot/chromeTests.py

```diff
@@ -323,25 +329,48 @@ def ARIAInvalid_spellingAndGrammar():
 	)
 
 
+def parseAndRun(instructionsFilePath):
+	lastSpeech = ''
+	file = open(instructionsFilePath, 'r', encoding='utf-8')
+	data = json.load(file)
+	for obj in data:
+		_builtIn.log(obj)
+		for command, args in obj.items():
+			if command == 'nav':
+				# TODO(zcorpan): this should open the file directly, not use iframe.
+				navPath = os.path.join(os.path.dirname(os.path.dirname(instructionsFilePath)), args[0]).replace('\\', '/')

```

Comment:
Backslash is invalid in a URL per https://url.spec.whatwg.org/#urls (though both work). I guess we don't have to do that, but wanted to avoid relying on URL error handling.

Reply:
I'll leave it in, because the theoretical problem with a non-Windows system and the path has a directory with a backslash in the name... still wouldn't work, because the browser's URL parser would change it to a forward slash. But I can add a comment.